.travis.yml

```diff
@@ -0,0 +1,20 @@
+sudo: required
+
+language: go
+
+services:
+  - docker
+
+script:
+- docker build -t aws-nuke --no-cache .
+- docker create --name aws-nuke aws-nuke
+- docker cp -L aws-nuke:/go/bin/aws-nuke .
+
+deploy:
+  provider: releases
+  api_key: $GITHUB_TOKEN
```

Comment:
I am currently testing with a new private one. Not sure how I can create one limited to this repo. Everything want full access to the whole organization (╯°□°）╯︵ ┻━┻